pkg/controllers/user/logging/utils/syslog.go

```diff
@@ -93,25 +92,6 @@ func newRFC5424Message(severityStr, app, token, msg string) []byte {
 	))
 }
 
-func getSeverity(severityStr string) syslog.Priority {
```

Comment:
What's the point to change code here? I don't see it used in your function.

Reply:
Yes, Windows node agent isn't related to `syslog`, however, `go test` is package-level, when running `go test -v -cover -tags 'test' ./pkg/agent/...`, it also search the dependencies of `./pkg/agent/cluster`.